app/views/content/train-to-be-a-teacher/_questions.html.erb

```diff
@@ -0,0 +1,28 @@
+
+<section class="category__questions">
+    <div class="category__question">
+        <h2>
+            <strong>What subject</strong> do you want to teach?
```

Comment:
I don't think this is the right heading if it is going to link to the SKE page (currently https://getintoteaching.education.gov.uk/improve-your-subject-knowledge)

SKE courses are more if you are concerned about your knowledge - and they may be a requirement of doing a course

That's not to say we shouldn't link to them from this page but I think we need to be more specific what this is about @valentine-carter